cli/packages/cmd/secrets.go

```diff
@@ -449,6 +494,18 @@ func getSecretsByNames(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if shouldExpand {
```

Comment:
bug: local secret reference isn't working 
<img width="1111" alt="Screenshot 2024-04-08 at 5 26 51 PM" src="https://github.com/Infisical/infisical/assets/9300960/709887f4-73f9-40dd-afd1-d485de21282d">
<img width="438" alt="Screenshot 2024-04-08 at 5 28 19 PM" src="https://github.com/Infisical/infisical/assets/9300960/0207df60-74a9-47e7-a5bd-cb4a169257a9">

Reply:
Unable to reproduce, can you include reproduction steps please?

![CleanShot 2024-04-12 at 13 48 52@2x](https://github.com/Infisical/infisical/assets/62331820/38e53ca6-7810-498d-b4f1-ee6662d03daa)

Reply:
Great catch! This is fixed now in this commit: https://github.com/Infisical/infisical/pull/1630/commits/71575b1d2ee49fb4103c91dca7fe06fc599de705